src/pages/understand-stacks/integrate-stacking-delegation.md

```diff
@@ -95,13 +99,37 @@ const delegetateResponse = await client.delegateStx({
 // }
 ```
 
-This method calls the [`delegate-stx`](/references/stacking-contract#delegate-stx) method of the Stacking contract.
+This method calls the [`delegate-stx`](/references/stacking-contract#delegate-stx) method of the Stacking contract. Note, that the amount can be higher or lower than the current account balance. Delegation does not yet lock the STX tokens, users can still transfer them.
 
 -> To avoid handling private keys, it is recommended to use the [Stacks Wallet](https://www.hiro.so/wallet) to sign the delegation transaction
 
-## Step 3: Stack STX tokens
+**Congratulations!** With the completion of this step, you successfully learnt how to use the Stacking library to ...
```

Comment:
could we just make this one sentence? One bullet point seems off because it's not really a list :)